docs/index.md

```diff
@@ -28,6 +28,7 @@ Freqtrade is a crypto-currency algorithmic trading software developed in python
 - Run: Run the bot on exchange with simulated money (Dry-Run mode) or with real money (Live-Trade mode).
 - Select markets: Create your static list or use an automatic one based on top traded volumes and/or prices (not available during backtesting). You can also explicitly blacklist markets you don't want to trade.
 - Download market data: Download historical data of the exchange and the markets your may want to trade with. The historical data can be based on [OHLCV](https://en.wikipedia.org/wiki/Open-high-low-close_chart) candles or be trade ticks (for exchanges that support this).
+- Strategy: Write your strategy in python, using [pandas](https://pandas.pydata.org/). Example strategies to inspire you are available in the [strategy repository](https://github.com/freqtrade/freqtrade-strategies).
```

Comment:
"using [pandas](https://pandas.pydata.org/), with indicators from the TA-LIB, qtpylib libraries, Freqtrade's own technical (link) repository or virtually any other python technical analysis library."

Reply:
well the order is a discussion point i think ...  to backtest your strategy -> test if the strategy may work - and eventually add more stuff - you'll need data to backtest.

Following your argumentation, then this should be first, and "Run:" should be last, since all other steps (apart from edge / plotting / analysis) should come before putting a bot to dry-mode...

Reply:
Yes, maybe first...

Reply:
At the beginning i have put numeration to see like a kind of step by step for the end user user experience (no matter there is no number but i think it it is important to go inside the end user head even it is not possible anymore :-) has we are not freqtrade virgin) and I think the whole doc sections should a kind of follow the end user workflow (this is of course not the subject for now). Sometimes the order depends of the way of developing a strategy so not so easy to define step by step :-).

And maybe `Develop a strategy` instead of `Strategy` like a kind of action verb

Reply:
A new inexperienced user first try to run the bot in the Dry-Run with a Sample strategy (or another one he finds available) and only then tries to modify it, backtest/adjust and then develop his own strategy... I guess...

Reply:
The order of features 'by importance' is more convenient, imho.

I we add there somewhere at the top 'no short trades', 'no custom position management', 'no pyramiding' (which are important limitations and, hence, are also important features), many people will read till these and quickly say 'oh, that's not for me'...

More important features first...

Reply:
> A new inexperienced user first try to run the bot in the Dry-Run with a Sample strategy (or another one he finds available) and only then tries to modify it, backtest/adjust and then develop his own strategy... I guess...

That's a bolt guess - and i think this hugely depends from the background of the user.

* If it's an experienced trader, he'll have a strategy in mind, and use the samples as "template" to develop his strategy, know that backtesting needs to come first - and only then dry/forward testing.
* if it's a ~developer~ non-trading experiened user new to trading - then yes, your workflow will probably what it is - but here we need to try and guide the user to FIRST backtest (even if a strategy is good for ETH stake, it may not work for different markets (USDT stake, for example)), and then dry-run.

> If we add there somewhere at the top 'no short trades', 'no custom position management', 'no pyramiding' 

I disagree ... the problem with having this list is that you'll always miss something.
You've listed 3 points - but what about hedging? trading non-crypto? trading futures?, taking partial profit? trading crypto options (yes, that thing exists)? 
 
I don't think we can possibly list all limitations here (they will be ever increasing as exchanges lauchn new features) - so i would not even start with this list to begin with - not on this page, anyway.
We're trying to "welcome" the user and show them the potential (being positive) ... not to tell him "go away, this is most likely not for you"...

Reply:
> And maybe `Develop a strategy` instead of `Strategy` like a kind of action verb

This is the list of *features*, why should it sound as an action?

I'd propose "Strategy Development:" there as the short 'name' (title) of the feature.